proxy/ptunnel/commands/agent.go

```diff
@@ -46,5 +52,18 @@ func RunAgentCommand(bf *client.BaseFlags, config *agentConfig) error {
 			fmt.Printf("PKS: %s Identities: %v ValidFor: %s \n", p.MD5, p.Principals, p.ValidFor.String())
 		}
 	}
+	if len(config.RemainingArgs) >= 1 {
```

Comment:
not sure it's worth the effort, but if we do it, better do it now. But if we make `enkit agent` take arbitrary argv to run in a shell, we won't be able to add subcommands, just like it happened with `enkit ssh`.

If we had:
`enkit agent list`
`enkit agent run`
`enkit agent eval`

corresponding to the options you have now, we could make `enkit agent run` automatically run a new shell, and `enkit agent run -- rsync -avz ...` run the specified command.

Thoughts?